resources/hu/cemu.po

```diff
@@ -173,6 +173,30 @@ msgstr "Tajvani"
 msgid "unknown"
 msgstr "ismeretlen"
 
+#: CemuApp.cpp
+msgid "AccountId missing (The account is not connected to a NNID)"
+msgstr "Hiányzó AccountId (Ez a fiók nincs NNID-hoz kapcsolva)"
+
+#: CemuApp.cpp
+msgid ""
+"IsPasswordCacheEnabled is set to false (The remember password option on your "
+"Wii U must be enabled for this account before dumping it)"
+msgstr ""
+"Az IsPasswordCacheEnabled értéke hamis (A jelszó megjegyzése opciónak "
+"(remember password) engedélyezve kell lennie Wii U-n képezés előtt)"
```

Comment:
I think it's important to preserve in the translation the fact that `IsPasswordCacheEnabled` is _set to false_ (here _false_ is meant as a boolean literal, referring to that particular value being present in a file or somewhere else).
Also is it needed to keep the original "remember password"? Isn't it clear enough from the translation?

Reply:
When translated back to English literally, „Az IsPasswordCacheEnabled értéke hamis” means ‘The value of IsPasswordCacheEnabled is false.’ I understand the confusion, since Google Translate, for some reason, ignores the word „értéke” (‘value’).
I left the phrase ‘remember password’ just in case, since the Wii U OS doesn't support Hungarian, so most people use it in English, but I can remove it if you want.

Reply:
Then it's ok for me to keep "remember password", didn't know that Nintendo didn't translate it into Hungarian.

My other objection was not on the verb you've chosen, but on the fact that the boolean literal "false" has been translated. "false" is the actual value of the flag IsPasswordCacheEnabled.

Reply:
Are you sure you want me to leave ‘false’ untranslated? We use „hamis,” even in IT.

Reply:
If the translated word for "false" is commonly used also for the boolean value, then it's fine to keep it as it is.